test/conftest.py

```diff
@@ -0,0 +1,130 @@
+import os
+import sys
+import datetime
+import pytest
+import snowflake.connector
+
+from contextlib import contextmanager
+from common_utils import delete_list_of_labels
+from configparser import RawConfigParser
+
+# Adds --profile option to pytest
+def pytest_addoption(parser):
+    parser.addoption(
+        "--profile",
+        action="store",
+        default="opscenter",
+        help="Connection profile name as specified in ~/.snowsql/config",
+    )
+
+
+# Returns value of --profile argument
+def snowsql_profile_name() -> str:
+    for i, arg in enumerate(sys.argv):
+        if arg == "--profile" and i + 1 < len(sys.argv):
+            return sys.argv[i + 1]
+    return "opscenter"
+
+
+def get_db_parameters_from_snowsql_config(profile):
```

Comment:
whynot use `deploy/helpers.py` for this?

Reply:
doh ... I don't know why I did not do it in the first place. Fixed. Thanks for pointing it out.